src/core/stream.c

```diff
@@ -195,10 +201,6 @@ QuicStreamFree(
     CxPlatDispatchLockUninitialize(&Stream->ApiSendRequestLock);
     CxPlatRefUninitialize(&Stream->RefCount);
 
-    if (Stream->ReceiveCompleteOperation) {
```

Comment:
where do we free this now?

Reply:
It's allocated with the Stream itself, so when the stream is freed.